neurom/view/common.py

```diff
@@ -381,11 +381,8 @@ def plot_style(fig, ax, **kwargs):
         plt.show()  # pragma no cover
 
 
-def plot_title(fig, ax, **kwargs):
-
-    """
-    Function that defines the title options
-    of a matplotlib plot.
+def plot_title(_, ax, **kwargs):
```

Comment:
each of the plot_ functions take the 'fig', but then never use it.  For not I have given it an un-named  arg, but I don't think we need to pass them in.

I would prefer to remove them for functions that don't use them.

Any comments @lidakanari, @eleftherioszisis